docs/cli-contract.md

```
@@ -0,0 +1,430 @@
+| `cmux [global-options] <command> [options]` | Run a named command. |
+| `cmux --help`, `cmux -h` | Print top-level usage without a socket. |
+| `cmux --version`, `cmux -v`, `cmux version` | Print version summary without a socket. |
+
+Global options:
+
+| Option | Contract |
+| --- | --- |
+| `--socket <path>` | Override the socket path for this invocation. |
+| `--password <value>` | Use an explicit socket password. Takes precedence over `CMUX_SOCKET_PASSWORD`. |
+| `--json` | Prefer machine-readable JSON output for commands that support it. |
+| `--id-format <refs|uuids|both>` | Select handle format in JSON and supported text output. |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)